src/app/roles.tsx

```diff
@@ -27,22 +27,20 @@ const pathsOnlyShownToStaffAndAbove = ["/lists"] as const;
 const pathsOnlyShownToAdmin = ["/admin"] as const;
 
 const getShownPagesByRole = (role: UserRole | null): readonly string[] => {
-    switch (role) {
-        case "admin":
-            return [
-                ...pathsShownToAllAuthenticatedUsers,
-                ...pathsOnlyShownToStaffAndAbove,
-                ...pathsOnlyShownToAdmin,
-            ];
-        case "manager":
-            return [...pathsShownToAllAuthenticatedUsers, ...pathsOnlyShownToStaffAndAbove];
-        case "staff":
-            return [...pathsShownToAllAuthenticatedUsers, ...pathsOnlyShownToStaffAndAbove];
-        case "volunteer":
-            return pathsShownToAllAuthenticatedUsers;
-        case null:
-            return pathsNotRequiringLogin;
+    if (adminRoles.includes(role as AdminRolesType)) {
```

Comment:
is there a way to get around this typecasting?

Reply:
It's not happy without - let me know if there are suggestions to get around it

Reply:
i'm not sure, if you can't thinking of anything maybe its ok to merge?